src/lib/panels-resize.js

```diff
@@ -21,28 +20,25 @@ const css = csjs`
   }
 `
 
-/*
-* opt:
-*   minWidth  : minimn width for panels
-*   x         : position of gutter at load
-*
-*
-*/
 export default class PanelsResize {
-  constructor (idpanel1, idpanel2, opt) {
-    var panel1 = document.querySelector(idpanel1)
-    var panel2 = document.querySelector(idpanel2)
-    this.panel1 = panel1
-    this.panel2 = panel2
-    this.opt = opt
+  constructor (panel) {
+    this.panel = panel
+    let self = this
+    const string = panel.style.minWidth
+    this.minWidth = string.length > 2 ? parseInt(string.substring(0, string.length - 2)) : 0
+    window.addEventListener('resize', function (event) {
```

Comment:
Use arrow function to have acces to `this`: 
`window.addEventListener('resize', (event) => this.setPosition(event))`